### PR TITLE
Hw02 express

### PR DESCRIPTION
**Pull request by erpua**
Check please

### CODE REVIEW

File: model/index.js [hunk 1]
```diff
@@ -1,15 +1,63 @@
...
+  return JSON.parse(data)
+}
+
+const listContacts = async () => {
+  return await readData()
+}
+
+const getContactById = async (id) => {
+  const data = await readData()
+  const [result] = data.filter((el) => el.id === id)
```
Comment (hold-opinov8) [inline model/index.js:18]: .find

File: model/index.js [hunk 2]
```diff
@@ -1,15 +1,63 @@
...
+  }
+  const data = await readData()
+  const newRecord = [...data, record]
+  await fs.writeFile(contactsPath, JSON.stringify(newRecord))
+  return record
+}
+
+const updateContact = async (id, body) => {
+  const data = await readData()
+  const [result] = data.filter((el) => el.id === id)
```
Comment (hold-opinov8) [inline model/index.js:54]: .find

**Review by hold-opinov8 — APPROVED** [review]

File: model/index.js (hunk 1 above)
Comment (Danil44) [inline model/index.js]: .find

File: model/index.js (hunk 2 above)
Comment (Danil44) [inline model/index.js]: .find

**Review by Danil44 — APPROVED** [review]

File: model/index.js (hunk 1 above)
Comment (erpua) [reply model/index.js]: Changed to find

File: model/index.js (hunk 2 above)
Comment (erpua) [reply model/index.js]: Changed to .method find

**Pull request merged**